AutomatedTesting/Gem/PythonTests/Atom/atom_utils/atom_constants.py

```diff
@@ -316,13 +316,58 @@ def depth_of_field(property: str = 'name') -> str:
             Use editor_entity_utils EditorEntity.add_components(list) to add this list of requirements.\n
           - 'Camera Entity' an EditorEntity.id reference to the Camera component required for this effect.
             Must be a different entity than the one which hosts Depth of Field component.\n
+          - 'CameraEntityId Override' Override enable for CameraEntityId, bool.
+          - 'Enabled Override' Override enable for Enabled, bool.
+          - 'QualityLevel Override' Override enable for QualityLevel, bool.
+          - 'ApertureF Override' Override enable for ApertureF, range 0.0-1.0.
+          - 'FocusDistance Override' Override enable for Focus Distance, range 0.0-1.0.
```

Comment:
nit: enable is sort of a bool action. this is an Override value for focus distance

Reply:
Fixed!